src/platform_linux/platform.h

```diff
@@ -88,14 +88,37 @@
 
 // Data unit constants
 #define KiB (1024UL)
-#define MiB (KiB * KiB)
-#define GiB (MiB * KiB)
+#define MiB (KiB * 1024)
+#define GiB (MiB * 1024)
+#define TiB (GiB * 1024)
 
+// Convert 'x' in unit-specifiers to bytes
 #define KiB_TO_B(x) ((x)*KiB)
 #define MiB_TO_B(x) ((x)*MiB)
 #define GiB_TO_B(x) ((x)*GiB)
+#define TiB_TO_B(x) ((x)*TiB)
+
+// Convert 'x' in bytes to 'int'-value with unit-specifiers
+#define B_TO_KiB(x) ((x) / KiB)
 #define B_TO_MiB(x) ((x) / MiB)
 #define B_TO_GiB(x) ((x) / GiB)
+#define B_TO_TiB(x) ((x) / TiB)
+
+// Return, as int, the fractional portion modulo a KiB for given x bytes.
+#define B_TO_KiB_FRACT(x)                                                      \
```

Comment:
This family of macros are used to generate the decimal portion of `bytes` when converted to a size-specifier unit.

Reply:
@ajhconway - In some other reviews you had said that we don't want to print decimal values using `%f` format. 

I forget now .. whether we are also not supposed to do division by float values, like is done in the body of the macros below.

I am hoping that this form of division to generate the decimal portion of the bytes value converted to a unit-specifier would be acceptable.

The meat of this change depends entirely on this small functionality.

Reply:
Yes! good catch! (Sorry, I was sloppy -- forgot my arithmetic.) Fixed in latest.